logos/benchmarks/benchmark_logos.py

```
@@ -3109,60 +3057,65 @@ def _ollama_compose_content(models_dir: str, local_models_dir: str) -> str:
+    os.makedirs(compose_dir, exist_ok=True)
+    compose_file = os.path.join(compose_dir, "docker-compose.yml")
+    with open(compose_file, "w") as fh:
+        fh.write(_sllm_head_compose_content(models_dir))
+    print(f"  [sllm] Starting head node (API port {_SLLM_API_PORT}) ...")
+    result = subprocess.run(
+        f"{sudo}docker compose -f {shlex.quote(compose_file)} up -d",
+        shell=True,
+    )
+    if result.returncode != 0:
+        raise RuntimeError(f"Failed to start SLLM head (exit {result.returncode}).")
+    print(f"  [sllm] Head node started.")
```

Ruff (0.15.18)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)